roles/openshift_openstack/tasks/unprovision.yml

```diff
@@ -1,4 +1,25 @@
 ---
+- name: Get kuryr services
+  delegate_to: "{{ groups.oo_first_master.0 }}"
+  oc_obj:
+    kind: endpoints
+    state: list
+    all_namespaces: true
+  register: svc_output
+
+- name: Delete kuryr LoadBalancers (cascading)
+  os_lbaas_deletion:
+    lbaas_annotation: "{{ item.metadata.annotations }}"
+  with_items: "{{ svc_output.results.results[0]['items'] }}"
+  when:
+    - ("{{ openshift_openstack_lbaasv2_provider }}" == "Octavia")
+    - not ("{{ item.metadata.name }}" == "kubernetes")
```

Comment:
The tox failures are related to using Jinja double brackets here. For some reason, the `when` sections don't need them and shouldn't be using them.

So it should look something like:

```yaml
  when:
    - openshift_openstack_lbaasv2_provider == "Octavia"
    - not (item.metadata.name" == "kubernetes"
```